docker-compose.yml

```diff
@@ -10,12 +10,22 @@ services:
     depends_on:
       - mariadb
       - memcache
```

Comment:
I wondered whether we should add mariatestdb here, but I guess it's not strictly needed immediately, just when you want to run the unit tests, so it's probablt appropriate not to.